gensim/models/indexer.py

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2013 Radim Rehurek <me@radimrehurek.com>
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
+
+
+"""
+Using 3rd party k-NN libraries to improve performance for method DocvecsArray.most_similar().
+More information can be found at: https://github.com/piskvorky/gensim/wiki/Ideas-&-Feature-proposals#integrate-a-fast-k-nn-library
+"""
+
+from doc2vec import Doc2Vec
+from word2vec import Word2Vec
+from annoy import AnnoyIndex
```

Comment:
@piskvorky You're right! I should have AnnoyIndexer implemented in a separate module right? 
Is a good idea to have a sub package named `indexer` in pacakge `gensim.models` ?